src/supporting-data/citations/config/CitationsColumnConfiguration.tsx

```diff
@@ -0,0 +1,125 @@
+import { Link } from 'react-router-dom';
+
+import { getEntryPathFor } from '../../../app/config/urls';
+
+import {
+  CitationsAPIModel,
+  CitationXRefDB,
+} from '../adapters/citationsConverter';
+import { ColumnConfiguration } from '../../../shared/types/columnConfiguration';
+import { JournalInfo } from '../../../shared/components/literature-citations/LiteratureCitation';
+import { Namespace } from '../../../shared/types/namespaces';
+
+export enum CitationsColumn {
+  // TODO: Report to backend that this field query is not working
+  authoringGroup = 'authoring_group',
+  doi = 'doi',
+  firstPage = 'first_page',
+  id = 'id', // Pubmed ID
+  journal = 'journal',
+  lastPage = 'last_page',
+  litAbstract = 'lit_abstract',
+  publicationDate = 'publication', // ⚠️ field doesn't contain the string "date"
+  // This map to a combination of all the fields to make a citation reference
+  reference = 'reference',
+  statistics = 'statistics',
+  title = 'title',
+  volume = 'volume',
+}
+
+// TODO: review
+export const defaultColumns = [
+  CitationsColumn.id,
+  CitationsColumn.title,
+  CitationsColumn.authoringGroup,
+  CitationsColumn.publicationDate,
+  CitationsColumn.journal,
+  CitationsColumn.firstPage,
+  CitationsColumn.lastPage,
+];
+
+export const primaryKeyColumn = CitationsColumn.id;
+
+const getEntryPath = getEntryPathFor(Namespace.citations);
+
+export const CitationsColumnConfiguration: ColumnConfiguration<
+  CitationsColumn,
+  Partial<CitationsAPIModel>
+> = new Map();
+
+// COLUMN RENDERERS BELOW
+CitationsColumnConfiguration.set(CitationsColumn.authoringGroup, {
+  label: 'Authors',
+  render: ({ citation }) => citation?.authors?.join(', '),
+});
+
+CitationsColumnConfiguration.set(CitationsColumn.doi, {
+  label: 'DOI',
+  render: ({ citation }) =>
```

Comment:
I would have expected too. But since PubMed is pointing to the entry and not the PubMed page I left it like that for now. Probably needs a round of UX review on what is expected for these columns